src/librustc/infer/error_reporting.rs

```diff
@@ -1295,7 +1296,9 @@ impl<'a, 'gcx, 'tcx> Rebuilder<'a, 'gcx, 'tcx> {
         let mut lifetimes = Vec::new();
         for lt in add {
             lifetimes.push(hir::LifetimeDef { lifetime: *lt,
-                                              bounds: hir::HirVec::new() });
+                                              bounds: hir::HirVec::new(),
+                                              pure_wrt_drop: false,
```

Comment:
Nit: Not that I care all that much, but the formatting here seems odd. I'd expect to either put the `}` on this line, or else move `lifetime:` to the next line, no?